test/apiv2/rest_api/test_rest_v2_0_0.py

```diff
@@ -50,23 +50,20 @@ class TestApi(unittest.TestCase):
     def setUp(self):
         super().setUp()
 
-        try:
-            TestApi.podman.run("run", "alpine", "/bin/ls", check=True)
-        except subprocess.CalledProcessError as e:
-            if e.stdout:
-                sys.stdout.write("\nRun Stdout:\n" + e.stdout.decode("utf-8"))
-            if e.stderr:
-                sys.stderr.write("\nRun Stderr:\n" + e.stderr.decode("utf-8"))
-            raise
+        TestApi.podman.run("run", "alpine", "/bin/ls", check=True)
+
+    def tearDown(self) -> None:
+        super().tearDown()
+
+        TestApi.podman.run("pod", "rm", "--all", "--force", check=True)
+        TestApi.podman.run("rm", "--all", "--force", check=True)
```

Comment:
Yeah, this is totally fine.  Crap can get left behind for any number of reasons, not all should cause test failures.  The time spent doing this will more than outweigh the aggravation and time of re-running the test b/c of flakes.